src/beam/locale/de/LC_MESSAGES/django.po

```diff
@@ -20,70 +20,235 @@ msgstr ""
 #: actions.py:80 themes/bootstrap4/templates/beam/partials/form_inline.html:78
 #: themes/bootstrap4/templates/beam/partials/form_inline.html:109
 #: themes/bootstrap4/templates/beam/partials/form_inline_tabular_row.html:59
-#: viewsets.py:203
+#: viewsets.py:210
 msgid "delete"
 msgstr "löschen"
 
-#: actions.py:97
+#: actions.py:100
 #, python-brace-format
 msgid "Deleted {count} {name}"
 msgstr "Es wurden {count} {name} gelöscht"
 
-#: actions.py:104
+#: actions.py:108
 msgid "update selected"
 msgstr "Auswahl bearbeiten"
 
-#: actions.py:145
+#: actions.py:149
 #, python-brace-format
 msgid "Updated {count} {name}"
 msgstr "Es wurden {count} {name} aktualisiert"
 
+#: contrib/auth/templates/registration/logged_out.html:5
+#, fuzzy
```

Comment:
Just as reminder: some translations are missing. Or are they all in django :)?

Reply:
Should all be in django